src/System.Runtime.Environment/src/RuntimeInfo.Linux.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Runtime.InteropServices
+{
+    public static class RuntimeInfo
```

Comment:
Is RuntimeInfo the best name for a static class? Many APIs in the framework that end with "Info" are classes that can be instantiated, e.g. FileInfo.